File: findMissingPosInt.cpp
// you can use includes, for example:
 #include <algorithm>
 #include <vector>
 #include <iostream> 
 using namespace std;

// you can write to stdout for debugging purposes, e.g.
// cout << "this is a debug message" << endl;

int solution(vector<int> &A) {
    sort(A.begin(), A.end());
    int smallest = 1;
    for(int i = 0; i < A.size(); i++){
        if(A[i] > smallest)
            return smallest;
        else if(smallest <= A[i])
            smallest++;
           
    }
}

File: findMissingPosInt.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(A):
    # write your code in Python 3.6
    x=1
    A.sort()
    for i in range(0,len(A)):
        if(A[i] > x):
            return x
        elif(x <= A[i]):
            x=x+1
    return x


    [1,1,3]